creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -316,7 +316,13 @@ router.get(
 )
 router.post(
   '/sync',
-  ensureStorageMiddleware,
+  // Force wipe syncs will free up storage space so we want to perform them regardless of current usage
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  async (req, res, next) => {
```

Comment:
async required?

Reply:
not technically, just thought it made sense to include because it returns a promise

Reply:
actually shouldnt it be return next() and await ensureStorageMiddleware()?

Reply:
idts because next and ensureStorageMiddleware() are both just promises as the only thing being returned. express awaits the promise that the middleware returns. so `await ensureStorageMiddleware()` would be redundant. does that make sense? also wdyt @dmanjunath ?

Reply:
ya i think it makes sense - i trust you

Reply:
yeah i don't think it's supposed to await the ensureStorageMiddleware because we dont want to call it as a function ourselves, we want express's middleware execution to continue and call it for us.

this is easy to test on stage. just set storage threshold for a node and try a sync without this body param and one sync with it and see if it goes through